app/src/test/java/mozilla/lockbox/presenter/AppLockedPresenterTest.kt

```diff
@@ -0,0 +1,185 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+@file:Suppress("DEPRECATION")
```

Comment:
🤔 I don't understand why this is here.

Reply:
The Kotlin warnings are now errors, so this is suppressing one of them